transport-native-epoll/src/main/c/io_netty_channel_epoll_Native.c

```diff
@@ -136,6 +151,21 @@ void throwOutOfMemoryError(JNIEnv* env) {
     (*env)->ThrowNew(env, exceptionClass, "");
 }
 
+int getSysctlValue(const char * property, int* returnValue) {
```

Comment:
yup that makes sense we dont have any other c file needing this function, I will make it static